event/generate.go

```diff
@@ -135,7 +135,7 @@ func main() {
 
```

Comment:
I suspect it's a good idea to update `nameMap` above so the generated names have proper casing:

https://github.com/gopherjs/vecty/blob/19a11ec3acc3ae6c2e095b1ace2b7134c5f28c10/event/generate.go#L22-L129

Reply:
Can you please expand on what you mean by proper casing? The goal of this PR is just to fix the selector to properly query the page. I believe other changes can be more meaningful in a separate PR.

Reply:
Sure. This PR currently adds some new functions to `event` package, like:

- `Devicechange`
- `Messageerror`
- `Slotchange`
- etc.

According to Go style conventions (https://golang.org/s/style#mixed-caps and https://golang.org/s/style#initialisms), identifiers should use MixedCaps casing. So the new functions should be named:

- `DeviceChange`
- `MessageError`
- `SlotChange`
- etc.

`nameMap` provides a mapping from HTML attribute names to proper Go style names. By adding new entries there, you can ensure the functions that are added as part of this PR follow Go style.

If your goal is not to add any new functions to `event` package as part of this PR, you'll need to make changes. That's not currently the case.

Reply:
Thanks for the explanation. I will add the mapping, it will be weird just submitting the changes in the generator file without the updated generated file.